telemetry/data_window.go

```diff
@@ -51,13 +60,34 @@ const (
 
 	maxSubWindowLength         = int(ReportInterval / SubWindowSize) // TODO: Ceiling?
 	maxSubWindowLengthInWindow = int(WindowSize / SubWindowSize)     // TODO: Ceiling?
+	promReadTimeout            = time.Second * 10
 )
 
 type windowData struct {
 	BeginAt        time.Time          `json:"beginAt"`
 	ExecuteCount   uint64             `json:"executeCount"`
 	TiFlashUsage   tiFlashUsageData   `json:"tiFlashUsage"`
 	CoprCacheUsage coprCacheUsageData `json:"coprCacheUsage"`
+	SQLUsage       sqlUsageData       `json:"SQLUsage"`
+}
+
+type sqlType struct {
+	Use      int64 `json:"Use"`
```

Comment:
Maybe better to use a hashmap instead of hard coding these sql types, since sql type is likely to be modified some day.

Reply:
Thank you for your commet, I will accept it! Hashmap is more resonable